sales_team_operating_unit/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 {
     "name": "Sales Team Operating Unit",
-    "version": "12.0.1.1.1",
+    "version": "12.0.1.1.2",
```

Comment:
could you rollback this please?, we manage it via ocabot. In any case this seems more an improvement than a patch. Thanks

Reply:
Sure, I will do it. Anyway, I'm interested on that. How is that ocabot working regarding version numbering?

Reply:
Thank you. It works by calling the bot and telling the bump version it has to do: [OCA BOT](https://odoo-community.org/blog/the-oca-blog-1/post/the-oca-github-bot-can-now-merge-pull-requests-92)

[About version numbers](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers)

Regards